Simplenote/Classes/NoteOptionsViewController.swift

```diff
@@ -1,14 +1,49 @@
 import UIKit
+import CoreSpotlight
 
 /// A class used to display options for the note that is currently being edited
-class NoteOptionsViewController: UITableViewController {
+final class NoteOptionsViewController: UITableViewController {
 
     /// Array of `Section`s to display in the view.
     /// Each `Section` has `Rows` that are used for display
     fileprivate var sections: [Section] {
         return [optionsSection, linkSection, collaborationSection, trashSection]
     }
 
+    /// The note from the editor that we will change settings for
+    private let note: Note
+
+    /// Determines if a note is currently awaiting publishing
+    /// changes to synchronise
+    private var noteIsChangingPublishState: Bool {
+        if (note.published && note.publishURL.isEmpty ||
+            !note.published && !note.publishURL.isEmpty) {
+            return true
+        }
+        return false
```

Comment:
Nipticky!

```
private var noteIsChangingPublishState: Bool {
    return (note.published && note.publishURL.isEmpty ||
                !note.published && !note.publishURL.isEmpty)
}
```

And you can take this one step further, and drop the `return` statement, since it's a single line, and returns its value implicitly.

Reply:
Thanks!